test/maps/port_space.wmf/scripting/test_portfield_blocked_by_rock.lua

```diff
@@ -0,0 +1,9 @@
+run(function()
+   --  This places to tower so that the port just fits the space.
+   map:place_immovable("greenland_rocks4", map:get_field(7,10))
+   local ship = create_infrastructure(59, 10)
+   game.desired_speed = 1000
+   wait_for_message("Land Ahoy!")
```

Comment:
Please extend with adding a rock to one of the neighbouring road fields and waiting for the portspace lost message.